views/index.pug

```diff
@@ -4,21 +4,21 @@ append head
   //- add page specific styles by appending to the head
   link(rel="stylesheet" href="/stylesheets/index.css")
   //- add page specific js
-  script(src="/javascripts/index.js" type="module" defer)
+  script(src="/javascripts/index.js" type="module" )
```

Comment:
Was `defer` causing problems here? Typically using `defer` is preferred so it does not pause the loading of the page.